### DISCUSSION TITLE
Database backend on server

### DISCUSSION THREAD

**Discussion by so010** [edumeet/edumeet#835] [Ideas]
needed for claim rooms and persistent room config
- mariaDB|postgres SQL?
- userTable (userID)
- userRooms (userID, roomName)
- roomTable (roomName, foreign key userID from userRooms,config:{'locked':false, 'pin':'',moderatorList:[],chatHistory:[]})

**Comment by misi** [comment]
IMHO it would be more flexible way to define an external API instead of pick one (or limited number of supported) database backends. Any backend db that implements this this API could be used to store roomnames or any further info we will need. 
UseCase example: We have own VC booking system already(for pexip), and so we have a roomname database, and I want to have uniq pexip and multiparty-meeting room names, and if we have two database it would be hard to sync rooms.
